boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -315,6 +316,18 @@ zephyr_udc0: &usbhs {
 	status = "okay";
 };
 
+&usdhc1 {
+	status = "okay";
+	pwr-gpios = <&gpio2 10 GPIO_ACTIVE_HIGH>;
+	cd-gpios = <&gpio2 9 GPIO_ACTIVE_LOW>;
+	mmc {
+		compatible = "zephyr,mmc-disk";
```

Comment:
I know about this and have a few reasons to leave it like this, which I don't feel the need to list because I was already considering just squashing the commit after it onto this one